curl_install.sh

```diff
@@ -33,7 +29,7 @@ get_latest_release() {
 # Install CNF-TestSuite
 LATEST_RELEASE=$(get_latest_release cncf/cnf-testsuite)
 mkdir -p ~/.cnf-testsuite
-curl -L https://github.com/cncf/cnf-testsuite/releases/download/$LATEST_RELEASE/cnf-testsuite-$LATEST_RELEASE.tar.gz -o ~/.cnf-testsuite/cnf-testsuite.tar.gz
+curl --silent -L https://github.com/cncf/cnf-testsuite/releases/download/$LATEST_RELEASE/cnf-testsuite-$LATEST_RELEASE.tar.gz -o ~/.cnf-testsuite/cnf-testsuite.tar.gz
 tar -C ~/.cnf-testsuite -xvf ~/.cnf-testsuite/cnf-testsuite.tar.gz
```

Comment:
```suggestion
tar -C ~/.cnf-testsuite -xf ~/.cnf-testsuite/cnf-testsuite.tar.gz
```
Maybe tar's `-v` option is what is showing the single line for cnf-testsuite? Hmm